src/redux/Restaurant/reducers.js

```diff
@@ -567,7 +567,10 @@ export default (state = initialState, action = {}) => {
       const printTask = state.ordersToPrint[orderId];
 
       if (!printTask) {
-        return state;
+        return {
+          ...state,
+          printingOrderId: null,
```

Comment:
My idea was that only `printFulfilled`/`printRejected` actions with the same order ID as in `printPending` should reset the state (assuming that printing is sequential)

Reply:
Did you see a situation when it was not working?